### ISSUE TITLE
0.17.5 dpkg installs as 0.17.4 - build error or typo?

### ISSUE THREAD

**Issue by the-moog** [jacobwhall/marktext#28]
### Description

The d/l for the 0.17.5 lists a .dpkg.  There is no version in the file name.  On installing it is reported as 0.17.4.  Either the version string was not updated or this is the wrong package file.

- [ x] Can you reproduce the issue? 

### Steps to reproduce

1. Click on asset in GitHub in a web-browser.
2. Download the file
3. Install
4. Examine list of installed packages

1. Uninstall Debian endorsed Flatpack as it's from 2022
2. Install GitHub version in the hope that one very annoying bug is gone.

**Expected behavior:**

Report installed version = latest version = 0.17.5

**Actual behavior:**

Reported version 0.17.4

**Link to an example: [optional]**

### Versions

- MarkText version:
- Operating system:

**Comment by jacobwhall** [comment]
Yep, this was just a typo. Apologies for the annoyance, I'll see about getting this resolved.
